extras/blank/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"zen_common"};
+        author = ECSTRING(main,Author);
+        authors[] = {};
```

Comment:
Maybe add a demo author? Otherwise, you could skip that while quickly going through.

Reply:
Not what that would be and I think leaving it empty is better than something like `authors[] = {"blank_author"};`.

Reply:
```suggestion
        authors[] = {"Username"};
```